src/components/input/DurationInput.js

```diff
@@ -1,12 +1,13 @@
 import React, { useRef } from 'react';
 import { Box, NumericInput } from '../../';
 import theme from './theme.css';
+import cx from "classnames";
 
 const transformToPaddedNumber = (number) => (number < 10 ? `0${number}` : number.toString());
 
 const MINUTES_STEP = 15;
 
-const DurationInput = ({ value, onChange, onBlur, onFocus, onKeyDown, autoFocus, textAlignRight, className }) => {
+const DurationInput = ({ value, onChange, onBlur, onFocus, onKeyDown, autoFocus, textAlignRight, className, error, max }) => {
```

Comment:
Q: How could we make it more clear the max is in minutes? `maxMinutes` could cause confusion.

Reply:
I think for this one an explanation in storybook would be the best solution
see https://github.com/teamleadercrm/ui/blob/next-release/src/components/input/Input.js#L42 -> https://components.teamleader.design/?path=/docs/4-low-level-blocks-form-elements-input--input

Though it seems only the main component in the story has this, that might be a configuration issue.

Otherwise you could maybe opt for `maxInMinutes`

Reply:
These both sound like reasonable solutions, I think I would prefer just going for the documentation option

Reply:
We've added a comment, but there seems to be something wrong with the docblocks rendering, will look at it in a different PR